packages/url_launcher/lib/url_launcher.dart

```diff
@@ -30,16 +33,14 @@ const MethodChannel _channel = MethodChannel('plugins.flutter.io/url_launcher');
 /// Note that if any of the above are set to true but the URL is not a web URL,
 /// this will throw a [PlatformException].
 ///
-/// [statusBarBrightness] is only used in iOS. Sets the status bar brightness
-/// of the application after opening a link. The previous value of the status
-/// bar is stored on the platform side and restored when returning to Flutter
-/// if used with `forceSafariVC` or on iOS version 10.0 and greater. Defaults
-/// to [Brightness.light] if unset, or does nothing if null is passed.
+/// [statusBarBrightness] Sets the status bar brightness of the application
+/// after opening a link on iOS. Defaults to [Brightness.light] and Does
+/// nothing if null value is passed.
 Future<void> launch(
   String urlString, {
   bool forceSafariVC,
   bool forceWebView,
-  Brightness statusBarBrightness = Brightness.light,
+  Brightness statusBarBrightness = Brightness.light
```

Comment:
Should this just be null by default? If the app adjusted their status bar brightness before, I would assume they would not expect a plugin to change it.